test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -982,4 +982,4 @@ View sys.synonyms
 View sys.syscharsets
 View sys.syslanguages
 View sys.xml_indexes
-View sys.xml_schema_collections
```

Comment:
Unwanted change. Dependency test are failing due to this diff.

Reply:
corrected!